src/documents/tests/test_tasks.py

```diff
@@ -103,6 +103,28 @@ def test_sanity_check_info(self, m):
         self.assertEqual(tasks.sanity_check(), "Sanity check exited with infos. See log.")
         m.assert_called_once()
 
+    def test_remove_expired_documents(self):
+        self.assertEqual(tasks.remove_expired_documents(), "0 expired documents deleted.")
+
+    def test_remove_expired_one_from_one(self):
+        Document.objects.create(title="test", content="my document", checksum="wow", added=timezone.now(),
+                                created=timezone.now(), modified=timezone.now(), expired=timezone.now()+timezone.timedelta(days=-1))
+        self.assertEqual(tasks.remove_expired_documents(), "1 expired documents deleted.")
+
+    def test_remove_expired_zero_from_one(self):
+        Document.objects.create(title="test", content="my document", checksum="wow", added=timezone.now(),
+                                created=timezone.now(), modified=timezone.now(), expired=timezone.now()+timezone.timedelta(days=1))
+        self.assertEqual(tasks.remove_expired_documents(), "0 expired documents deleted.")
+
+    def test_remove_expired_one_from_three(self):
+        Document.objects.create(title="test", content="my document", checksum="wow1", added=timezone.now(),
+                                created=timezone.now(), modified=timezone.now(), expired=timezone.now()+timezone.timedelta(days=-1))
+        Document.objects.create(title="test", content="my document", checksum="wow2", added=timezone.now(),
+                                created=timezone.now(), modified=timezone.now(), expired=timezone.now()+timezone.timedelta(days=1))
+        Document.objects.create(title="test", content="my document", checksum="wow3", added=timezone.now(),
+                                created=timezone.now(), modified=timezone.now())
+        self.assertEqual(tasks.remove_expired_documents(), "1 expired documents deleted.")
+
```

Comment:
This tests to check for an string return. But I think they also should test, that the documents actually are removed.